Factory/AbstractFactory/VipAccountFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Factory
+{
+    class VipAccountBuilder : IAccountBuilder
```

Comment:
And again. This PR about Factory, right?

Reply:
Rewrite